### PR TITLE
tox 3.0.0 (new formula)

### PR DESCRIPTION
**Pull request by amancevice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tox.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Tox < Formula
+  include Language::Python::Virtualenv
+
+  desc "tox automation project"
```
Comment (fxcoudert) [inline Formula/tox.rb]:
```
  * C: 4: col 9: Description should start with a capital letter
  * C: 4: col 9: Description shouldn't start with the formula name
```

File: Formula/tox.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  def install
+    virtualenv_create(libexec, "python3")
+    virtualenv_install_with_resources
+  end
+
+  test do
+    ENV["LC_ALL"] = "en_US.UTF-8"
+    (testpath/"tox.ini").write "[tox]"
+    assert_match "usage", shell_output("#{bin}/tox --help")
```
Comment (fxcoudert) [inline Formula/tox.rb]: We will need a test that exercises some of the functionality…

**Review by fxcoudert — DISMISSED** [review]

**Comment by amancevice** [comment]
Requests changes made, @fxcoudert

**Comment by ilovezfs** [comment]
```
    virtualenv_create(libexec, "python3")
```
is actually handled automatically so I've removed it

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for the 🆕 formula @amancevice! 🚀

**Pull request closed without merging**